### PR TITLE
tests: not fail the post operations and send slack notifications

### PR DESCRIPTION
**Pull request by cpanato**
fixes: INST-808

### CODE REVIEW

**Comment by cpanato** [comment]
ok to test

File: Jenkinsfile [hunk 1]
```diff
@@ -522,3 +530,16 @@ def reportStatusToGithub(status, context, commitId) {
...
 }
+
+def notifyBuildSlack(String message) {
+  if(message.contains("master") || message.contains("track-1") || message.contains("log-es-error")) {
+    def colorCode = '#FF0000'
+    def subject = "Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]': ${message}"
+    def summary = "${subject} (${env.BUILD_URL})"
+
+    // Send notifications
+    slackSend(color: colorCode, message: summary, channel: "#team-installer")
```
Comment (enxebre) [inline Jenkinsfile]: This is probably ok to start, but experience tells me that it's going to end up being to noisy and we'll move it to a dedicated channel

Comment (cpanato) [reply Jenkinsfile]: this will send just when master/track-1 fail as we have today

**Comment by cpanato** [comment]
@enxebre @mxinden can you please take a look? thanks

File: Jenkinsfile [hunk 2]
```diff
@@ -396,19 +395,31 @@ pipeline {
   post {
     always {
       node('worker && ec2') {
-        forcefullyCleanWorkspace()
```
Comment (mxinden) [inline Jenkinsfile]: What is the reason for removing this line?

File: Jenkinsfile [hunk 3]
```diff
@@ -522,3 +533,16 @@ def reportStatusToGithub(status, context, commitId) {
     """
   }
 }
+
+def notifyBuildSlack(String message) {
+  if(message.contains("master") || message.contains("track-1") || message.contains("log-es-error")) {
```
Comment (mxinden) [inline Jenkinsfile]: If an `log-es-error` happens on a PR, we still want to be notified, right?

File: Jenkinsfile (hunk 2 above)
Comment (cpanato) [reply Jenkinsfile]: forgot to readd, will put it back

File: Jenkinsfile (hunk 3 above)
Comment (cpanato) [reply Jenkinsfile]: yes

**Comment by cpanato** [comment]
@enxebre @mxinden can you please take a look? thanks

**Pull request merged**